docs/scss/_blog.scss

```diff
@@ -0,0 +1,11 @@
+.ov-masthead--blog {
+    background-image: url("/images/oVirt_Blog_BG_Overlay.png")left top no-repeat var(--ov--color-brand--100);
+    .ov-heading--lg {
```

Comment:
I wonder if these styles need to be page specific. Could we just have it so it targets it with `.ov-masthead .ov-heading--lg`? then we don't need to repeat it for the different pages.

Reply:
That sounds like a good idea to me since the headers on some of the pages are different heights.